yarn.lock

```diff
@@ -13143,6 +14129,15 @@ websocket@1.0.29:
     typedarray-to-buffer "^3.1.5"
     yaeti "^0.0.6"
 
+"websocket@git://github.com/frozeman/WebSocket-Node.git#browserifyCompatible":
```

Comment:
No chance in hell I'm approving this PR with this package in it :angry:

Reply:
These also came from the `ganache-core-coverage` fork